src/App/Pages/Settings/SettingsPage/SettingsPageViewModel.cs

```diff
@@ -295,7 +295,8 @@ public void BuildList()
                 autofillItems.Add(new SettingsPageListItem
                 {
                     Name = AppResources.AutofillAccessibilityService,
-                    SubLabel = _deviceActionService.AutofillAccessibilityServiceRunning() ?
+                    SubLabel = (_deviceActionService.AutofillAccessibilityServiceRunning() 
```

Comment:
I would make this into a variable above the `.Add()` so that we can keep the shorthand operator clean and fix the weird indenting:

```
var accessibilityEnabled = _deviceActionService.AutofillAccessibilityServiceRunning() &&
    _deviceActionService.AutofillAccessibilityOverlayPermitted();
```